tools/issue-labeler/src/IssueLabelerService/Properties/ServiceDependencies/gh-issue-labeler - Zip Deploy/appInsights1.arm.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json#",
```

Comment:
Not a concern, nor overly important, but any chance we can scrub the number from the file name?